### DISCUSSION TITLE
Help me with your choice of songs in 2024/好きな曲を教えて下さい

### DISCUSSION THREAD

**Discussion by hoshinokohana** [NAIST-SE/AttractingContributors#1170] [2024 Attraction Contributions]
I want to collect everyone's best choice of songs for this year, any genre would be welcomed.
For example, my personal choice for 2024 right now would be Girls Never Die from tripleS, which is a kpop song.
Please tell me your choice about your best selection!

**Comment by hoshinokohana** [comment]
https://github.com/hoshinokohana/AttractingContributors/tree/hoshinokohana-patch-1
I've created one branch on this pr, please feel free to leave your choice~

**Reply by wangzaixiaoliunai** [reply]
help me https://github.com/wangzaixiaoliunai/AttractingContributors-musical_instruments.git
